commands/list.go

```diff
@@ -47,12 +62,13 @@ func newListCmd() *cobra.Command {
 	c.Flags().BoolVar(&jsonFormat, "json", false, "List models in a JSON format")
 	c.Flags().BoolVar(&openai, "openai", false, "List models in an OpenAI format")
 	c.Flags().BoolVarP(&quiet, "quiet", "q", false, "Only show model IDs")
+	c.Flags().StringVar(&backend, "backend", "", "Specify the backend to use (llama.cpp, openai)")
```

Comment:
```suggestion
	c.Flags().StringVar(&backend, "backend", "", "Specify the backend to use ("+
		strings.Join(slices.Collect(maps.Keys(ValidBackends)), ", ")+")")
```

Even add a function to return the keys as it'd be used in at least 3 places - `validateBackend`, flag for list, flag for run.

```
func ValidBackendsKeys() []string {
	return slices.Collect(maps.Keys(ValidBackends))
}
```

Reply:
addressed [here](https://github.com/docker/model-cli/pull/126/commits/b3044ee4b28fb1ec3ee69a89dac5e942a85cee16)